docs/includes/mac-cache-folders.md

```diff
@@ -8,12 +8,14 @@ You can clear the cache by using the personality menu of any task pane add-in.
 
     ![Screenshot of clear web cache option on personality menu.](../images/mac-clear-cache-menu.png)
```

Comment:
Is the personality menu available in Outlook Mac? If not, we may want to clarify that in the lead-in statements.

Reply:
I'm not sure. I will merge this PR and then follow up on this question separately. Created a [work item](https://office.visualstudio.com/OC/_workitems/edit/5898246) to track the task.